run_benchmark_serve.sh

```
@@ -0,0 +1,146 @@
+set -ex
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)